### PR TITLE
HBASE-22729 Start RSGroupInfoManager as default

### PR DESCRIPTION
**Pull request by Apache9**
Amending-Author: Duo Zhang <zhangduo@apache.org>

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 41 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ HBASE-22514 Compile Tests _ |
| +1 | mvninstall | 324 | HBASE-22514 passed |
| +1 | compile | 61 | HBASE-22514 passed |
| +1 | checkstyle | 84 | HBASE-22514 passed |
| +1 | shadedjars | 304 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 38 | HBASE-22514 passed |
| 0 | spotbugs | 261 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 259 | HBASE-22514 passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 323 | the patch passed |
| +1 | compile | 61 | the patch passed |
| +1 | javac | 61 | the patch passed |
| +1 | checkstyle | 83 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 288 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 977 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 36 | the patch passed |
| +1 | findbugs | 283 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 9915 | hbase-server in the patch failed. |
| +1 | asflicense | 30 | The patch does not generate ASF License warnings. |
| | | 13510 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.master.replication.TestTransitPeerSyncReplicationStateProcedureRetry |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/555 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux e766f3832c53 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-555/out/precommit/personality/provided.sh |
| git revision | HBASE-22514 / 548e8c4ed8 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/1/testReport/ |
| Max. process+thread count | 4597 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfo.java [hunk 1]
```diff
@@ -34,21 +32,38 @@
 public class RSGroupInfo {
   public static final String DEFAULT_GROUP = "default";
   public static final String NAMESPACE_DESC_PROP_GROUP = "hbase.rsgroup.name";
+  public static final String TABLE_DESC_PROP_GROUP = "hbase.rsgroup.name";
```
Comment (saintstack) [inline hbase-common/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfo.java]: Its intentional this and the define above have same value?

Comment (Apache9) [reply hbase-common/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfo.java]: Yes, intentional. We can use the same name in namespace and table properties, the only problem is that the old name contains a 'NAMESPACE' and it is IA.Public, so we need to introduce a new 'TABLE' one.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 77 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ HBASE-22514 Compile Tests _ |
| +1 | mvninstall | 372 | HBASE-22514 passed |
| +1 | compile | 62 | HBASE-22514 passed |
| +1 | checkstyle | 96 | HBASE-22514 passed |
| +1 | shadedjars | 315 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 39 | HBASE-22514 passed |
| 0 | spotbugs | 298 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 296 | HBASE-22514 passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 442 | the patch passed |
| +1 | compile | 82 | the patch passed |
| +1 | javac | 82 | the patch passed |
| +1 | checkstyle | 101 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 375 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1430 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 50 | the patch passed |
| +1 | findbugs | 329 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 16858 | hbase-server in the patch failed. |
| +1 | asflicense | 28 | The patch does not generate ASF License warnings. |
| | | 21503 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestFromClientSide |
|   | hadoop.hbase.client.TestSnapshotTemporaryDirectoryWithRegionReplicas |
|   | hadoop.hbase.client.TestFromClientSideWithCoprocessor |
|   | hadoop.hbase.client.TestFromClientSide3 |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/555 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 4e0b5d6c80bf 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-555/out/precommit/personality/provided.sh |
| git revision | HBASE-22514 / 78af5b70e6 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/3/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/3/testReport/ |
| Max. process+thread count | 4939 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-555/3/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**